osu.Game/Graphics/UserInterface/OsuButton.cs

```diff
@@ -79,13 +77,5 @@ protected override bool OnMouseUp(MouseUpEvent e)
             Content.ScaleTo(1, 1000, Easing.OutElastic);
             return base.OnMouseUp(e);
         }
-
-        protected override SpriteText CreateText() => new OsuSpriteText
-        {
-            Depth = -1,
-            Origin = Anchor.Centre,
-            Anchor = Anchor.Centre,
-            Font = OsuFont.GetFont(weight: FontWeight.Bold)
```

Comment:
How is this unused?

Reply:
I was also wondering, may I ask the answer? I was trying to understand where is this `OsuButton` used but could not get into a breakpoint. My idea would be that this is the button players click while playing the game, and it does not have to be any text on it because it depends on texture / skin.

Reply:
I believe it was left over from a previous implementation but is no longer called anywhere.

Reply:
It's called by the base class no?

Reply:
...hmm... I'll have to reassess this. The usages I could see of `OsuButton` were adding their own text locally.